wcivf/apps/elections/models.py

```diff
@@ -376,6 +376,14 @@ def is_pcc(self):
         """
         return self.ballot_paper_id.startswith("pcc")
 
+    @property
+    def is_consituency(self):
+        return self.ballot_paper_id.startswith("gla.c", "senedd.c", "sp.c")
```

Comment:
`startswith` needs to be passed a tuple, so `self.ballot_paper_id.startswith(("gla.c", "senedd.c", "sp.c"))` (double braces).

I suspect this silently failing in the template layer at the moment - can you add a test for these properties please? (same for `is_regional` below)